tests/model_forms/tests.py

```diff
@@ -2854,6 +2854,11 @@ def test_custom_field_with_queryset_but_no_limit_choices_to(self):
         f = StumpJokeWithCustomFieldForm()
         self.assertEqual(f.fields['custom'].queryset, 42)
 
+    def test_fields_for_model_applies_limit_choices_to(self):
+        fields = fields_for_model(StumpJoke, ['has_fooled_today'])
+        self.assertIn(self.threepwood, fields['has_fooled_today'].queryset)
```

Comment:
I'd use `self.assertSequenceEqual(fields['has_fooled_today'].queryset, [self.threepwood])` so two assertions aren't required. I'll change the other cases separately.